mix.exs

```diff
@@ -38,7 +29,7 @@ defmodule ExMagick.Mixfile do
     [
       app: :exmagick,
       version: "1.0.0",
-      elixir: "~> 1.2",
+      elixir: "~> 1.3",
```

Comment:
What do you thing about update elixir to last version?

https://elixir-lang.org/blog/2018/01/17/elixir-v1-6-0-released/

Reply:
This is the minimum requirement.

There's no reason for us to require the most recent elixir version.

We can, however, do as we did here to require a version with OTP/20 to use the dirty scheduler